grammar.js

```diff
@@ -71,6 +71,8 @@ module.exports = grammar({
     "#if",
     "#else",
     "#endif",
+    "class",
```

Comment:
There's also "struct end", not sure if you want to include it in the same change set

Reply:
Yes, I plan on adding all the "x .. end" like structures once I get the parsing right for this one 😃